science/gmt5/Portfile

```diff
@@ -30,9 +36,18 @@ use_xz              yes
 distname            gmt-${version}
 distfiles           ${distname}-src${extract.suffix}
 
-checksums           rmd160  900befd66ec4a9aea75c53d99ed83f7e25163e35 \
-                    sha256  078d4997507cb15344c74a874568985e45bdbd6d3a72d330c74c47f4c0359bb1 \
-                    size    59175704
+# checksums for gmt5
+set rmd160(gmt5)    900befd66ec4a9aea75c53d99ed83f7e25163e35
+set sha256(gmt5)    078d4997507cb15344c74a874568985e45bdbd6d3a72d330c74c47f4c0359bb1
+set size(gmt5)      59175704
+# checksums for gmt6
+set rmd160(gmt6)    c9d276100c5e78191ccbabd2ce084798a1f2046f
+set sha256(gmt6)    940ed0c49f18cd749edbe65ab84b28e2f6775b4529a71e9a8efd67bf94241b76
+set size(gmt6)      65172968
+
+checksums           rmd160  $rmd160(${subport}) \
+                    sha256  $sha256(${subport}) \
+                    size    $size(${subport})
```

Comment:
Please use the standard formatting for the checksums so they are easy to update moving forward using:

`sudo port -v checksum
`

You need to put them into an `if {${subport} eq "gmt5"}` block.

Reply:
Thanks. I've committed the change in https://github.com/macports/macports-ports/pull/5250/commits/27fb86f99a96ff0149a9e0cc1aa01cc0a63eabdc